types/router.d.ts

```diff
@@ -104,6 +104,14 @@ export interface RouteConfig {
   pathToRegexpOptions?: PathToRegexpOptions
 }
 
+export type RouteConfig = BaseRouteConfig & ({
```

Comment:
can you instead create 2 interfaces `RouteConfigSingleView` and `RouteConfigMultipleViews` that extend the base config and export RouteConfig as the type union?